plugins/outputs/nebius_cloud_monitoring/nebius_cloud_monitoring_test.go

```diff
@@ -119,6 +119,31 @@ func TestWrite(t *testing.T) {
 				w.WriteHeader(http.StatusOK)
 			},
 		},
+		{
+			name:   "label with name 'name' is replaced with 'label_name'",
+			plugin: &NebiusCloudMonitoring{},
+			metrics: []telegraf.Metric{
+				testutil.MustMetric(
+					"cluster",
+					map[string]string{
+						"name": "accounts-daemon.service",
+					},
+					map[string]interface{}{
+						"value": 9226,
+					},
+					time.Unix(0, 0),
+				),
+			},
+			handler: func(t *testing.T, w http.ResponseWriter, r *http.Request) {
+				message, err := readBody(r)
+				require.NoError(t, err)
+				require.Len(t, message.Metrics, 1)
+				require.Equal(t, "cluster_value", message.Metrics[0].Name)
+				require.Equal(t, "accounts-daemon.service", message.Metrics[0].Labels["_name"])
```

Comment:
Shouldn't the check better be if `_name` label exists?

Reply:
That's what is done, isn't it?

Reply:
No, as it now will panic or error out if `message.Metrics[0].Labels` doesn't have an item called `_name`.